src/components/simple-color-picker/simple-color-picker.style.ts

```diff
@@ -1,10 +1,15 @@
 import styled, { css } from "styled-components";
 
+import { ValidationProps } from "../../__internal__/validations";
 import StyledValidationIcon from "../../__internal__/validations/validation-icon.style";
 
 const BORDER_WIDTH = 2;
-const getRoundedMaxWidth = (maxWidth, childWidth) =>
-  Math.floor(maxWidth / childWidth) * childWidth;
+
+type MaxWidth = string | number;
+type ChildWidth = string | number;
```

Comment:
is there any reason to use 2 separate types here? Why not just `type CssWidth = string | number` (or whatever name you prefer, I'm not suggesting that's the best name, just the first one I thought of) and use it for both arguments/props?